fontbe/src/features/marks.rs

```diff
@@ -55,10 +55,17 @@ struct MarkLookupBuilder<'a> {
     mark_glyphs: BTreeSet<GlyphName>,
 }
 
+/// Abstract over the difference in anchor shape between mark2lig and mark2base/mark2mark
+#[derive(Debug, Clone, PartialEq)]
+enum BaseOrLigaAnchors<T> {
```

Comment:
Suggest spelling out Ligature somewhere. Enum and variant names seem duplicative. AnchorType::Ligature?

I'd rather we write out Ligature in full but if we must shorten it lets at least pick one shortening. Right now we have lig and liga.